packages/synapse-react-client/src/components/RecentPublicationsGrid/RecentPublicationsGrid.tsx

```diff
@@ -178,14 +205,23 @@ function RecentPublicationsGrid(props: RecentPublicationsGridProps) {
               sx={{
                 whiteSpace: 'nowrap',
                 alignSelf: 'flex-start',
+                boxShadow: 'none',
```

Comment:
With great care, I think `!important` helps here because none of the variants (:hover/:active) should have boxShadow

```suggestion
                boxShadow: 'none !important',
```